classes/main.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python3
+
+import time
+
+'''
+
+In this game we have two role - invader and hero. Both can bullet.
+
+Invaders are located at the top of game field and always move from the right side to the left and revert.
+Also they slowly move to the bottom.
+
+Main hero is located at the centre at the bottom. He can move to the left and to the right.
+If he bullet some invader, his bonus is grow.
+When invader bullet to main hero, the main hero's life is decrease.
+
+'''
+
+
+class GameScreen(object):
+
+    def __init__(self, filename, height, width):
+        self.image = filename
+        self.height = height
+        self.width = width
+
+
+class Unit(object):
+
+    def __init__(self, filename, x, y, bonus=0, speed=5):
+        self.image = filename
+        self.x = x
+        self.y = y
+        self.width = 10  # temporary when we don't have real images
+        self.height = 10  # must be height of real image
+        self.bonus = bonus
+        self.speed = speed
+        self.is_dead = False
+        self.bullet = Bullet(self)
+        self.shift = 10
+
+    def check_explosion(self, other_unit):
+        pass
```

Comment:
no sense in adding all that abstract method since we even don't know how to use them and whether we really need that methods.

If you define an abstract method you'd better do "raise NotImplemented" instead of pass